docs/repo-deb.md

```diff
@@ -0,0 +1,66 @@
+# DEB repository server
+
+| | repo-deb01.osuosl.theforeman.org |
+| - | - |
+| type | OpenStack VM |
+| OS | CentOS Stream 9 |
+| CPUs | 2 |
+| RAM | 4GB |
+| Storage | /dev/sda (30GB): root, /dev/sdb (150GB): data (LVM) |
+| Managed by | [deb.pp](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/profiles/manifests/repo/deb.pp) |
+
+## Domains
+
+These domains are all hosted on the server.
+
+* deb.theforeman.org
+* stagingdeb.theforeman.org
+* archivedeb.theforeman.org
+
+### Fastly CDN
+
+A Fastly CDN exists that sits in front of:
+
+* deb.theforeman.org
+* stagingdeb.theforeman.org
+* archivedeb.theforeman.org
+
+For these, the server acts as a backend while the content is served from the Fastly CDN to users.
```

Comment:
Maybe mention the whole backend configuration? It's not as obvious to me when reading this the design where fastly serves the domains but we have those related backends configured. I know this is noted in the backend section, but it's inferred instead of stated the design.

Reply:
I've updated repo-deb.md, mind having a look? If that's fine I'll adjust the other two files to the same structure

Reply:
yes looks good!